### PR TITLE
Doc: Changelog for 0.5.3

### PR DESCRIPTION
**Pull request by andythenorth**

### CODE REVIEW

File: docs/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,16 @@
+0.5.3 (2020-07-05)
+------------------------------------------------------------------------
+This release primarily restores legacy industry properties and vars which were removed in NML 0.5.0.  This is done for compatibility, so that older industry grfs can still be compiled with NML 0.5.3.  
+
+See the NML 0.5.0 changelog entry for a list of the legacy industry properties and vars that were removed, and are restored in NML 0.5.3.
+
+- Change: Show more info in `nmlc --version
+- Change: Tweak construction of git identifiers
+- Codechange: Reintroduce 0.4 syntax for compatibility
```
Comment (LordAro) [inline docs/changelog.txt:9]:
```suggestion
- Change: Reintroduce 0.4 industry property syntax for compatibility
```

File: docs/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,16 @@
+0.5.3 (2020-07-05)
+------------------------------------------------------------------------
+This release primarily restores legacy industry properties and vars which were removed in NML 0.5.0.  This is done for compatibility, so that older industry grfs can still be compiled with NML 0.5.3.  
+
+See the NML 0.5.0 changelog entry for a list of the legacy industry properties and vars that were removed, and are restored in NML 0.5.3.
+
+- Change: Show more info in `nmlc --version
+- Change: Tweak construction of git identifiers
```
Comment (LordAro) [inline docs/changelog.txt:8]: Remove

File: docs/changelog.txt [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+- Codechange: Reintroduce 0.4 syntax for compatibility
+- Codechange: Make id_list callbacks take a 'name' argument
```
Comment (LordAro) [inline docs/changelog.txt:10]: Remove

File: docs/changelog.txt [hunk 4]
```diff
@@ -1,3 +1,16 @@
...
+- Fix: Procedure calls reference counting
```
Comment (LordAro) [inline docs/changelog.txt:11]: Remove

File: docs/changelog.txt [hunk 5]
```diff
@@ -1,3 +1,13 @@
+0.5.3 (2020-07-05)
+------------------------------------------------------------------------
+This release primarily restores legacy industry properties and vars which were removed in NML 0.5.0.  This is done for compatibility, so that older industry grfs can still be compiled with NML 0.5.3.  
+
+See the NML 0.5.0 changelog entry for a list of the legacy industry properties and vars that were removed, and are restored in NML 0.5.3.
```
Comment (FLHerne) [inline docs/changelog.txt:5]: ... A warning will be shown when these are used, and they may be removed again in some future version.

File: docs/changelog.txt [hunk 6]
```diff
@@ -1,3 +1,13 @@
+0.5.3 (2020-07-05)
+------------------------------------------------------------------------
+This release primarily restores legacy industry properties and vars which were removed in NML 0.5.0.  This is done for compatibility, so that older industry grfs can still be compiled with NML 0.5.3.  
+
+See the NML 0.5.0 changelog entry for a list of the legacy industry properties and vars that were removed, and are restored in NML 0.5.3. A warning will be shown when these are used, and they may be removed again in some future version.
+
+- Change: Show more info in `nmlc --version
+- Codechange: Reintroduce industry property 0.4 syntax for compatibility
+
```
Comment (FLHerne) [inline docs/changelog.txt:9]: - Fix: Rounding errors for some speed values (#147)

File: docs/changelog.txt [hunk 7]
```diff
@@ -1,3 +1,13 @@
+0.5.3 (2020-07-05)
+------------------------------------------------------------------------
+This release primarily restores legacy industry properties and vars which were removed in NML 0.5.0.  This is done for compatibility, so that older industry grfs can still be compiled with NML 0.5.3.  
+
+See the NML 0.5.0 changelog entry for a list of the legacy industry properties and vars that were removed, and are restored in NML 0.5.3. A warning will be shown when these are used, and they may be removed again in some future version.
+
+- Change: Show more info in `nmlc --version
+- Codechange: Reintroduce industry property 0.4 syntax for compatibility
```
Comment (FLHerne) [inline docs/changelog.txt:8]: Just "Change"?  Or even "Add"

**Review by FLHerne — APPROVED** [review]
Thanks!

Let's do this, releases are cheap whenever #155 is ready.

**Pull request merged**